src/util/Constants.js

```diff
@@ -602,8 +602,11 @@ exports.VerificationLevels = createEnum(['NONE', 'LOW', 'MEDIUM', 'HIGH', 'VERY_
  * * UNKNOWN_REDISTRIBUTABLE
  * * UNKNOWN_GIFT_CODE
  * * UNKNOWN_GUILD_TEMPLATE
+<<<<<<< HEAD
  * * UNKNOWN_INTERACTION
  * * UNKNOWN_APPLICATION_COMMAND
+=======
+>>>>>>> 70b04688 (fix: fix bugs found in runtime)
```

Comment:
Hello there git rebase

Reply:
I am very confused how I didn't get an error when rebasing...git you okay?